docs/_content/ontwikkelaars/algemeen/notificaties.md

```diff
@@ -100,3 +91,153 @@ Bronnen:
 * [Digilevering koppelvlakspecificatie](https://www.logius.nl/sites/default/files/public/bestanden/diensten/DigiLevering/Koppelvlakspecificatie.pdf)
 * [Digilevering documentatie](https://www.logius.nl/diensten/digilevering/documentatie)
 * [Regie- en Zaakservices specificaties](https://www.gemmaonline.nl/images/gemmaonline/c/cd/Koppelvlakspecificatie_Regie-zaak_services_v1.0.pdf)
+
+# API specificatie (concept)
+
+[Voorbeeld sequence diagram (in-depth)](http://sequencediagram.org/index.html#initialData=C4S2BsFMAIDkHtQDMQGNICdIDsBQuAHAQw1FRGO2GgC0AlAYUJLIqKrkRBVSNBgbwAtgXjYcwZqTRsOdIgCMFYALIBFKa0rUAIo3z0GAWgB8CZGj4gBw0eKoAuaAAUA8gGUAKtAD0RAiA+AG4AjD4KGGL2ADrYPj4ADADMAEwhRgBeRADWOPG45tyW-NCCImISpvJKqmpODFh8MMDwAaix8clpmTl5PgVcPFY25fbAADxGRtXKwOpOAJI6TiEpSbiGk0aFQyVldhJOABKens7QKQkhHT4AQpHY0ADmmJBPIADOwFiYACbQQkg1CWKzW+VwAF4IQtsGAQERwJ9htAguxoAAqdGGTHQIhIF4PX5Q-B6YxmQbFaylWwVRwuDzePwBYJhRRRSCAqg4G5ddJZXJxfo7SkjA5UKqKWbzaDuACuCg+qAwIAUMFJ0BaGtaaB5qT5vUFAwsvD2NLGWxmtScrgA0rhSVthSaqftacBjqdzpdroKAIIKdmc6hEdgvIQh7LAcFQmFwhFIkqox6Y0k4vEEsREiEGRimJ3I11jJxuLy+fyBULhB72HyrJI+bAU52QD43W6YNAAC0k+dNo0qJktczqLkiv1l6ABLY+RBeWugIOgdaNRWb1P7VAtkqt0FtG0YjqbBbNh2gJzOFwSCRuADV4OAWvAQL98kP1KYTKTiwyy8zK43jWGD4fDEYAkxebA2w7VBu3yIA)
+
+## Notificatie API specificatie
+
+Wanneer een abonnement wordt aangemaakt, gelden de volgende vereisten:
+
+1. Berichten dienen 7 dagen bewaard te blijven (`messageRetentionDuration=604800`)
+2. Abonnementen dienen actief te blijven voor 31 dagen bij geen activiteit (`expirationPolicy.ttl=2678400`)
+3. Abonnementen mogen worden opgezegd als het pushUrl 7 dagen of langer niet bereikbaar is geweest.
+4. Opgehaalde berichten mogen niet meer dan 1x worden opgehaald (pull) of worden afgeleverd (push). Oude berichten worden dus verwijderd (`retainAckedMessages=false`)
+
+### Bronnen (topics) beheren
+
+Ondersteuning voor: GET (``list`` en ``retrieve``), POST, DELETE
+
+**/api/v1/bronnen[/{id}]**
+```javascript
+{
+  "url": "https://example.com/api/v1/bronnen/9b441a" // Alleen-lezen
+  "id": "9b441a",
+  "rsin": "082096752011",
+  "naam": "zaken"
+}
+```
+
+*Gebaseerd op Google Cloud Pub/Sub*
+
+projects.topics.create/delete/get/list
+
+### Abonnementen (subscriptions)
+
+Ondersteuning voor: GET (``list`` en ``retrieve``), POST, PUT, PATCH en DELETE
+
+**/api/v1/abonnementen[/{id}]**
+```javascript
+{
+  "url": "https://example.com/api/v1/api/v1/abonnementen/761a81", // Alleen-lezen
+  "id": "761a81", // Alleen-lezen, eventueel een UUID als onderliggende techniek dit toestaat
+  "bron": {  // De "bron" vorm het topic: <rsin>-<naam>
```

Comment:
moet dit een lijst worden van bronnen? of een `bron.rsin` met een lijst van topics? of verwacht je dat er voor elk ding een subscription aangemaakt wordt?

Ik zie een beetje een directory voor me als:

| Bron | RSIN | Sub? |
|------|------|------|
| ZRC  | 1234 | [x]  |
| DRC  | 1234 | [x]  |
| BRC  | 1234 | [ ]  |

Dit binnen 1 organisatie.

Als je latere complexere filters wil configureren (zoals je in RabbitMQ zou doen), dan weet ik niet meteen wat handiger is. Waarschijnlijk nu om complexiteit laag te houden inderdaad 1 bron per subscription doen.